Integrations/integration-Awake_Security.yml

```diff
@@ -0,0 +1,607 @@
+        # scores mean more unusual behavior
+        #
+        # Our product does not yet assign meaning
+        # to these scores since unusual behavior is
+        # not necessarily malicious, but empirically
+        # partioning the scores by quartile appears to
+        # map closely enough to Demisto's classification
+        # scheme for now.
+        if percentile <= 25:
+            # Typical range for people doing ordinary
+            # office work
+            score = 0
+        elif percentile <= 50:
+            # People doing something out of the ordinary
+            # compared to others (i.e. performing weird
+            # searches, learning something exotic, making
+            # an unusual purchase)
+            score = 1
+        elif percentile <= 75:
+            # Something doing multiple things out of the
+            # ordinary, worth investigating
+            score = 2
+        else:
+            # Probably bad or at least not compliant with
+            # company policy.  You have to make an effort
+            # to score this high.
+            score = 3
+
+        return { "Vendor": "Awake Security", "Type": indicator_type, "Indicator": lookup_key, "Score": score }
+
+    def lookup(lookup_type, lookup_key):
+        path = "/lookup/" + lookup_type
+
+        request["lookup_key"] = lookup_key
+
+        if "lookback_minutes" in args:
+            request["lookback_minutes"] = int(args["lookback_minutes"])
+        else:
+            request["lookback_minutes"] = 480
+
+        response = requests.post(prefix + path, json=request, headers=headers, verify=verify)
+
+        return response.json()
+
+    def lookupDevice(lookup_key):
+        contents = lookup("device", lookup_key)
+
+        humanReadableFields = [
+            "deviceScore",
+            "deviceName",
+            "deviceType",
+            "os",
+            "osVersion",
+            "commonEmail",
+            "commonUsername",
+            "tags",
+            "recentIP",
+            "activeIP",
+            "nSimilarDevices",
+            "ipCount",
+            "applicationCount",
+            "protocols",
+            "firstSeen",
+            "lastSeen",
+        ]
+
+        if "deviceScore" in contents:
+            dbotScore = toDBotScore("device", contents["deviceScore"], lookup_key)
+        else:
+            dbotScore = None
+
+        humanReadable = displayTable([contents], humanReadableFields)
+
+        returnResults(contents, humanReadable, dbotScore)
+
+    def lookupDomain(lookup_key):
+        contents = lookup("domain", lookup_key)
+
+        humanReadableFields = [
+            "notability",
+            "isAlexaTopOneMillion",
+            "isDGA",
+            "intelSources",
+            "numAssociatedDevices",
+            "numAssociatedActivities",
+            "approxBytesTransferred",
+            "protocols",
+            "firstSeen",
+            "lastSeen",
+        ]
+
+        if "notability" in contents:
```

Comment:
i'll check it thrilly when we'll have access, but i think dbotscore 0 might be a better option here